ui/package.json

```diff
@@ -41,7 +41,7 @@
     "react-dom": "^18.3.1",
     "react-draggable": "^4.4.6",
     "react-redux": "^8.0.7",
-    "react-router-dom": "^6.23.1",
+    "react-router-dom": "^6.24.0",
```

Comment:
please do a quick smoke test for this... react router sometimes releases breaking changes in minor/patch revisions